dm/dm/master/scheduler/scheduler.go

```diff
@@ -2232,6 +2194,8 @@ func (s *Scheduler) handleWorkerOffline(ev ha.WorkerEvent, toLock bool) error {
 // pulling binlog using relay or not is determined by whether the worker has enabled relay.
 func (s *Scheduler) tryBoundForWorker(w *Worker) (bounded bool, err error) {
```

Comment:
should update function comment

Reply:
addressed in https://github.com/pingcap/tiflow/pull/5077/commits/37b8c98aabc4ec518b994540958adf3729375e73